src/index.test.js

```diff
@@ -1,5 +1,3 @@
-/* eslint ava/no-only-test:0, ava/no-skip-test:0 */
-import test from 'ava'
```

Comment:
That would be great. I reeeeally love AVA's testing API and would love to keep it if possible. 👍